packages/components/tests/dummy/app/templates/components/stepper.hbs

```diff
@@ -115,25 +113,34 @@
     '
   />
   <p class="dummy-paragraph">Renders to:</p>
-  <Hds::Stepper::Indicator::Step @text="1" @status="incomplete" @isInteractive={{true}} />
-  <h5 class="dummy-h5">Notating status</h5>
+  <Hds::Stepper>
+    <Hds::Stepper::Indicator::Step @text="1" @status="incomplete" @isInteractive={{true}} />
```

Comment:
This is one place where, if we intend for the user to nest an interactive element, we could provide an example: 

```suggestion
    <Hds::Stepper::Indicator::Step @text="1" @status="incomplete" @isInteractive={{true}}><a href="#">Step One</a></Hds::Stepper::Indicator::Step>
```